modules/querying/partials/histogram-api.adoc

```diff
@@ -0,0 +1,127 @@
+=== Compute histogram statistics
+
+`POST :14240/gsqlserver/gsql/stats/histogram`
+
+This endpoint computes histogram statistics for a specified attribute of a vertex type.
+
+==== Parameters
+
+
+|===
+|Parameter |Description |Data type
+
+|`graph`
+|Name of the graph.
+Required.
+|`STRING`
+
+
+|`vertex`
+|Name of the vertex type.
+Required.
+|`STRING`
+
+|`attribute`
+|Name of the attribute.
+Required.
+|`STRING`
+
+|`bucket`
+|The number of intervals (https://en.wikipedia.org/wiki/Data_binning[buckets]) to divide the range of attribute values by.
+Required.
+
+It is recommended that each bucket has at least 100 values.
+Therefore, a good rule of thumb is to divide the number of vertices by 100 to decide the number of buckets.
+For example, if the vertex type has 100,000 vertices, set this parameter to 1000.
```

Comment:
I would say at least 100 values. So instead of recommending that you set the parameter to `1000` exactly, we should recommend to set it no greater than `1000`. And then we should probably add that 256 is typically a standard default.